code/datums/station_traits/neutral_traits.dm

```diff
@@ -386,6 +386,8 @@
 		for(var/obj/effect/landmark/start/ai/secondary/secondary_ai_spawn in GLOB.start_landmarks_list)
 			secondary_ai_spawn.latejoin_active = TRUE
 
+#define SKUB_IDFC "i don't frikkin' care"
```

Comment:
Thanks for putting in the idea but could you change the random option to be less ambigiously the same as neutral? 
If you want a funny message it could be "I'm on the best side" or something